independent-publisher-connectors/Informa/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1976 @@
+                        "type": "string",
+                        "description": "Encoded CIF",
+                        "title": "Encoded CIF"
+                      },
+                      "denominacion": {
+                        "type": "string",
+                        "description": "Designation",
+                        "title": "Designation"
+                      },
+                      "campoCodificadoProvincia": {
+                        "type": "object",
+                        "properties": {
+                          "valor": {
+                            "type": "string",
+                            "description": "Value",
+                            "title": "Value"
+                          },
+                          "tablaDecodificacion": {
+                            "type": "string",
+                            "description": "Decoding Table",
+                            "title": "Decoding Table"
+                          }
+                        },
+                        "description": "campoCodificadoProvincia"
+                      },
+                      "municipio": {
+                        "type": "string",
+                        "description": "Municipality",
+                        "title": "Municipality"
+                      }
+                    }
+                  },
+                  "description": "datosProducto"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/api/get-product": {
+      "get": {
+        "summary": "Get Product Information",
+        "description": "This service displays a list of the different available products and the product code.",
+        "operationId": "InformaGetProduct",
+        "parameters": [
+          {
+            "name": "cif",
+            "in": "query",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "CIF/NIF",
+            "description": "Tax Identification Number (CIF) of the company or Tax Identification Number (NIF) of the self-employed individual from whom the information is requested.",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "product",
+            "default": "INFORME_CONECTOR",
+            "in": "query",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "Product",
+            "description": "Product ID",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "language",
+            "default": "es",
+            "in": "query",
+            "type": "string",
+            "required": false,
+            "x-ms-summary": "Language",
+            "description": "Language in which to receive the decoding of literals.",
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Success", "Bad request", etc.

Reply:
ok